CMakeLists.txt

```diff
@@ -121,12 +191,20 @@ target_sources(viam-ur
     src/viam/ur/module/ur_arm_state_independent.cpp
     src/viam/ur/module/utils.cpp
 )
+
+# Tell boost::dll to use std::filesystem rather than boost::filesystem.
+target_compile_definitions(viam-ur
```

Comment:
I think, technically, we maybe are supposed to add boost::headers as a library below? I'll check.